packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_flavor_config/__brick__/android/environments/production/google-services.json

```diff
@@ -7,7 +7,7 @@
   "client": [
     {
       "client_info": {
-        "mobilesdk_app_id": "Make sure you add your own Firebase production config.",
+        "mobilesdk_app_id": "Make sure you add your own Firebase development config.",
```

Comment:
I guess here we need to keep the previous version, considering this is a production config ?